examples/v1beta1/windows2022.yaml

```diff
@@ -0,0 +1,33 @@
+# This example NodePool will provision instances running Windows Server 2022
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.